frontend/public/components/conditions.tsx

```diff
@@ -1,43 +1,51 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
-
+import { Table } from './factory';
 import { Timestamp } from './utils';
 import { CamelCaseWrap } from './utils/camel-case-wrap';
 
 export const Conditions: React.SFC<ConditionsProps> = ({conditions}) => {
-  const rows = _.map(conditions, (condition, i) => <div className="row" key={i}>
-    <div className="col-xs-4 col-sm-2 col-md-2">
-      <CamelCaseWrap value={condition.type} />
-    </div>
-    <div className="col-xs-4 col-sm-2 col-md-2">
-      {condition.status}
-    </div>
-    <div className="hidden-xs hidden-sm col-md-2">
-      <Timestamp timestamp={condition.lastUpdateTime || condition.lastTransitionTime} />
-    </div>
-    <div className="col-xs-4 col-sm-3 col-md-2">
-      <CamelCaseWrap value={condition.reason} />
-    </div>
-    {/* remove initial newline which appears in route messages */}
-    <div className="hidden-xs col-sm-5 col-md-4 co-break-word co-pre-line">
-      {_.trim(condition.message) || '-'}
-    </div>
-  </div>);
+  const ConditionsTableHeader = () => {
+    return [
+      {
+        title: 'Type',
+      },
+      {
+        title: 'Status',
+      },
+      {
+        title: 'Updated',
+      },
+      {
+        title: 'Reason',
+      },
+      {
+        title: 'Message',
+      },
+    ];
+  };
+
+  const ConditionsTableRows = ({componentProps}) => {
+    return _.map(componentProps.data, (condition) => {
+      return [
+        { title: <CamelCaseWrap value={condition.type} /> },
```

Comment:
this should currently match the default pf4 table behavior (w/ any CSS overrides we have for `pf-c-table`). I am not explicitly overriding column widths for `Table` at the moment though, so the default pf4 responsive behavior would apply.

https://pf4.patternfly.org/components/Table/examples/
https://pf4.patternfly.org/components/Table/examples-full/?component=Simple%20table

Reply:
> this should currently match the default pf4 table behavior (w/ any CSS overrides we have for `pf-c-table`).

So equal width columns? We might look at this since many of our tables have columns with data significantly wider than other columns. Nodes page comes to mind (image name and size).

Reply:
Please see the "Table w/ width modifiers" example:
https://pf4.patternfly.org/components/Table/examples/

`.pf-m-width-[10, 15, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, or 90]`, that is the default functionality for PF4 table width modifiers at the moment. 

The existing Bootstrap column width modifiers you have should also work similarly to PF's `.pf-u-w-25{-on-[breakpoint]}` modifiers, however I don't think you'd want to use mobile modifiers now that we are stacking here and using the default PF4 mobile behavior.
https://pf4.patternfly.org/utilities/Sizing/examples/
 
We should probably file anything upstream in PF if we determine something else is needed on this.

Reply:
so we could pass:
 ``` 
{ title: <CamelCaseWrap value={condition.type} />, props: { className: 'pf-m-width-10` },
```
or
```
{ title: <CamelCaseWrap value={condition.type} />, props: { className: 'col-md-6` },
```
for example...